include/circt-c/ExportLLVM.h

```diff
@@ -0,0 +1,27 @@
+//===-- circt-c/ExportLLVM.h - C API for exporting LLVM IR --------*- C -*-===//
```

Comment:
nit: emacs tag is nod needed anymore 
```suggestion
//===----------------------------------------------------------------------===//```